src/REPLMode/argument_parsers.jl

```diff
@@ -100,7 +100,7 @@ function parse_package_args(args::Vector{PackageToken}; add_or_dev=false)::Vecto
         elseif modifier isa Rev
             pkg.rev = modifier.rev
         else
-            pkgerror("Package name/uuid must precede subdir specifier `[$arg]`.")
+            pkgerror("Package name/uuid must precede subdir specifier `$args`.")
```

Comment:
Here, the issue was that there isn't any `arg` variable. I wasn't completely sure of what the intended argument should have been here - it could've also been `args[1]`?